test/errors-adjacency-matrix-modifications.js

```diff
@@ -163,7 +163,7 @@ describe('errors-adjacency-matrix-modifications', function () {
       test.noErrors('test-files/adjacency-matrix-modifications/missing-data-output.xlsx');
     })
   })
-  
+
```

Comment:
I know that technically, semicolons are optional in JavaScript, but I suggest that we still try to insert them where they are supposed to appear (e.g., after function calls).